packages/web-components/src/components/leadspace/__stories__/leadspace.stories.ts

```diff
@@ -147,18 +142,18 @@ export const CenteredWithImage = ({ parameters }) => {
           `;
         })}
       </dds-button-group>
-      <dds-image slot="image" class="bx--image" alt="${ifNonNull(alt)}" default-src="${leadspaceImg}">
-        <dds-image-item media="(min-width: 672px)" srcset="${leadspaceImg}"></dds-image-item>
-        <dds-image-item media="(min-width: 0)" srcset="${leadspaceImg}"></dds-image-item>
-      </dds-image>
+      <dds-leadspace-image slot="image" class="bx--image" alt="${ifNonNull(alt)}" default-src="${image}">
```

Comment:
since we're using this new `leadspace-image` component, could you update the example in the README too?

Reply:
Fixed!